Firestore/core/src/firebase/firestore/util/async_queue.h

```diff
@@ -22,7 +22,9 @@
 #include <functional>
 #include <memory>
 #include <mutex>  // NOLINT(build/c++11)
+#include <vector>
 
+#include "Firestore/core/src/firebase/firestore/immutable/append_only_list.h"
```

Comment:
Is this used?

Reply:
So sorry to leave all these unused imports! I've been spoiled by Android and Web's linters.